skimage/transform/radon_transform.py

```diff
@@ -153,6 +153,11 @@ def iradon(radon_image, theta=None, output_size=None,
         Assume the reconstructed image is zero outside the inscribed circle.
         Also changes the default output_size to match the behaviour of
         ``radon`` called with ``circle=True``.
+    freqcutoff: float, optional (default 1, ie., no cutoff)
```

Comment:
To me, these are mistakes and I had in mind that the standard is to do not specify values if there is signature inspection.
I base my judgement on this: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
and we follow numpy's standards.